### PR TITLE
Add Predictive Back Implementation with latest alpha libraries

### PR DESCRIPTION
**Pull request by tiwiz**
This PR updates latest Activity Libraries to support the predictive back, and adds a summary of the different canonical layouts for easier reachability

### CODE REVIEW

File: CanonicalLayouts/README.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ For more information see the [Large screen canonical layouts](https://developer.
 
 Clone the [user-interface-samples](https://github.com/android/user-interface-samples) repository, then open the canonical layout samples as projects in Android Studio.
 
+## Available Canonical layouts and related implemtations
```
Comment (romanofranz) [inline CanonicalLayouts/README.md:11]: Typo: Implementations

**Review by romanofranz — CHANGES_REQUESTED** [review]
Please fix the typo. Rest is fine, thanks!

File: CanonicalLayouts/README.md (hunk 1 above)
Comment (tiwiz) [reply CanonicalLayouts/README.md]: ✅  Done

**Review by romanofranz — APPROVED** [review]
LGTM!Thanks!

**Pull request merged**